scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -72,7 +73,9 @@ task SplitIntervals {
   Int disk_memory = if (defined(split_intervals_mem_override)) then split_intervals_mem_override else 16
   Int java_memory = disk_memory - 4
 
-  String gatkTool = if (defined(interval_weights_bed)) then 'WeightedSplitIntervals' else 'SplitIntervals'
+  String gatkTool = if (use_interval_weights && defined(interval_weights_bed)) then 'WeightedSplitIntervals' else 'SplitIntervals'
```

Comment:
Do we need to have both an optional `File? interval_weights_bed` and `Boolean use_interval_weights`? Could the exome branch just force in an empty value for `interval_weights_bed` instead?

Reply:
so forcing an existing value to null didn't seem to work that great for me when I attempted it in wdl.  That's why I called this solution hacky and said I'd do a better one later.  Is there an `undefined` object I'm supposed to use instead of `null`?  A lot of this particular patch is me working around the fact that I can't just drive that File object to null when I want to, tbh.  But it's works, however, clunkily, for now.

Reply:
If only there were a literal for `null` / `none` in WDL 1.0... [this](https://github.com/broadinstitute/gatk/blob/d1907a813360472b1d8e3d827bf991d43a699a98/scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl#L29-L32) seems to be the best practice.

Reply:
Same question - and comment below - I think the optional `interval_weights_bed` should be enough and you shouldn't need the boolean.

Reply:
Using that hack, which I didn't think would work for File objects, did work.  That allowed me to clean up the implementation!